src/umd/ReactDOMUMDEntry.js

```diff
@@ -11,24 +11,24 @@
 
 'use strict';
 
+var React = require('React');
```

Comment:
I don't think so—don't we requite `React` all over the place in `ReactDOM`? For example `ReactCompositeComponent` (now part of `ReactDOM`) uses `React.isValidElement`. I think @sebmarkbage made it work during the package split.. somehow. Maybe [with this](https://github.com/facebook/react/blob/df0d2a5106a225ae94ec978a99342443d57f83c2/grunt/config/browserify.js#L18).